pkg/controller/kieapp/constants/constants.go

```diff
@@ -76,23 +76,23 @@ var VersionConstants = map[string]*api.VersionConfigs{
 		BrokerImage:      "amq-broker-73-openshift",
 		BrokerImageTag:   "7.3",
 		DatagridImage:    "datagrid73-openshift",
-		DatagridImageTag: "1.1",
+		DatagridImageTag: "1.2",
 	},
 	LastMicroVersion: {
 		APIVersion:       v1.SchemeGroupVersion.Version,
 		ImageTag:         "1.1",
 		BrokerImage:      "amq-broker-73-openshift",
 		BrokerImageTag:   "7.3",
 		DatagridImage:    "datagrid73-openshift",
-		DatagridImageTag: "1.1",
+		DatagridImageTag: "1.2",
 	},
 	LastMinorVersion: {
 		APIVersion:       v1.SchemeGroupVersion.Version,
 		ImageTag:         "1.0",
 		BrokerImage:      "amq-broker-73-openshift",
 		BrokerImageTag:   "7.3",
 		DatagridImage:    "datagrid73-openshift",
-		DatagridImageTag: "1.1",
+		DatagridImageTag: "1.2",
```

Comment:
same, but for 7.4.0 ... as a reference, here's where we define which version is which...
https://github.com/kiegroup/kie-cloud-operator/blob/32ea6118a75f11c9d48362578127608f10e8efa6/pkg/controller/kieapp/constants/constants.go#L9-L16

Reply:
This mapping helps it make sense. Thanks again.